tests/tools.vitruv.applications.transitivechange.tests/src/tools/vitruv/applications/pcmumlclassjava/LinearTransitiveChangeTest.xtend

```diff
@@ -0,0 +1,17 @@
+package tools.vitruv.applications.pcmumlclassjava
+
+import org.junit.jupiter.api.BeforeAll
+
+/**
+ * This class allows to create transitive change tests that to do use the fully network,
+ * only the linear network <code>PCM <--> UML <--> Java</code>. This is required, as the
+ * used applications contain stil some issues regarding the fully transitive use case.
+ * TODO TS: these issues should be fixed in the PCM/UML/Java applications and all sub classes should extend TransitiveChangeTest directly
+ */
+abstract class LinearTransitiveChangeTest extends TransitiveChangeTest {
+
+    @BeforeAll
+    def static void enforceLinearNetwork() {
+        linearNetwork = true
+    }
+}
```

Comment:
This is the workaround which allows that some test to only use the linear network, while the test whose issues were completely fixed still use the fully connected network.